lib/danger/scm_source/git_repo.rb

```diff
@@ -19,6 +19,26 @@ def diff_for_folder(folder, from: "master", to: "HEAD")
       self.log = repo.log.between(from, to)
     end
 
+    def renamed_files
+      before_name_regexp = /rename from (.*)rename/
+      after_name_regexp = /rename to (.*)$/
```

Comment:
These regexes aren't correct or restrictive enough (you'll end up matching file contents as well).
Should use a single match with multiple groups like this: `/^diff --git a\/(.*?) b\/(.*?)/`